config/charts/inferencepool/Chart.yaml

```diff
@@ -7,3 +7,9 @@ type: application
 version: 0.0.0
 
 appVersion: "0.0.0"
+
+dependencies:
+  - name: jaeger
+    version: "2.11.0"
```

Comment:
Jaeger version 2.12.0 has been released perhaps you should move to that release?